frontend/src/routes/OrganizationInjector.tsx

```diff
@@ -0,0 +1,30 @@
+import { Suspense } from 'react';
+import { Navigate, Outlet } from 'react-router-dom';
+import { injectOrganizationToRequest } from '../apis/axios';
+import Spinner from '../components/@shared/Spinner';
+import { ROUTE_PATH } from '../constants/routes';
+import { useGetOrganizations } from '../hooks/@queries/organization';
+
+const OrganizationInjector = () => {
+  return (
+    <Suspense fallback={<Spinner />}>
+      {(() => {
+        const { organizations } = useGetOrganizations();
+
+        const lastAccessedOrganization = organizations.find(
+          organization => organization.lastAccessed
+        );
+
+        if (lastAccessedOrganization) {
+          injectOrganizationToRequest(String(lastAccessedOrganization.organizationId));
+
+          return <Outlet />;
+        }
+
+        return <Navigate to={ROUTE_PATH.JOIN_ORGANIZATION} />;
+      })()}
+    </Suspense>
+  );
+};
```

Comment:
컴포넌트를 즉시 실행으로 리턴 후 Suspense로 감싸줬습니다. 컴포넌트 두개 생성안하고 Suspense로 감싸주고 싶어서 이렇게 해봤습니다.
이건 좀 호불호가 갈릴 수도 있을 것 같습니다.

```suggestion
const OrganizationInjector = () => {
  return (
        const { organizations } = useGetOrganizations();

        const lastAccessedOrganization = organizations.find(
          organization => organization.lastAccessed
        );

        if (lastAccessedOrganization) {
          injectOrganizationToRequest(String(lastAccessedOrganization.organizationId));

          return <Outlet />;
        }

        return <Navigate to={ROUTE_PATH.JOIN_ORGANIZATION} />;
      }
};
```
OrganizationInjector를 이렇게만 하고 route.tsx에서 
```jsx
<Route element={<Suspense fallback={<Spinner/>}>
                <OrganizationInjector />
              </Suspense>}/>
```
이런식으로 사용해볼까 했는데 추상화 수준이 안맞아서 혼자 뚱뚱한 느낌이더라고요. 보시고 어떤게 더 읽기 좋은지 얘기 나눠보면 좋을 것 같아요.
개인적으로는 현재 방식은 route를 깔끔하게 유지하는 것 같고, 후자는 OrganizaionInjector의 가독성을 높이는 것 같습니다.